utils/merge.py

```
@@ -1,91 +1,174 @@
+        "path_format": existing_settings.output_scheme,
+    }
+
+
+def fetch_audible_metadata(asin: str, api_url: str = "https://api.audnex.us") -> dict:
+    """Fetch book metadata from Audible API using ASIN."""
+    try:
+        response = requests.get(f"{api_url}/books/{asin}", timeout=30)
+        response.raise_for_status()
+        return response.json()
+    except requests.RequestException as e:
+        logger.error(f"Failed to fetch metadata for ASIN {asin}: {e}")
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)